flame/utils/mock_flame_core.py

```
@@ -182,6 +182,7 @@ def send_message_and_wait_for_responses(self,
     def submit_final_result(self,
                             result: Any,
                             output_type: Literal['str', 'bytes', 'pickle'] = 'str',
```

Ruff (0.14.14)

[warning] Unused method argument: `output_type`

(ARG002)